en/services/multi_gcs.md

```diff
@@ -0,0 +1,128 @@
+# Multi GCS protocol
+## Introduction
```

Comment:
FYI, need to change the "autopilot" focus if we allow the `CONTROL_STATUS` to be emitted by other components.